src/dapp-tests/integration/tests.sh

```diff
@@ -2,15 +2,56 @@
 
 set -e
 
-export SOLC_FLAGS="--evm-version constantinople"
-export DAPPSYS_PATH="lib"
+# clean up
+trap 'killall geth && rm -rf "$TMPDIR"' EXIT
```

Comment:
It's sad that we will end up killing any instance of geth running on this host. Ideally we would be a bit more surgical about it.

Reply:
Ideally... But `dapp testnet` starts geth as another subprocess so we can't use `$!`. I'm not sure what another way to do it would be

Reply:
Namespaces/cgroups could be a way, or otherwise process groups. I started looking into the latter but then it became a rabbit hole of itself, and the former seems a bit overkill, so I let it go.